### PR TITLE
Movies_List

### PR DESCRIPTION
**Pull request by stunnerboyone**
- [DEMO LINK](https://stunnerboyone.github.io/react_movies-list-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,98 +1,14 @@
 /* eslint-disable max-len */
 
 import './App.scss';
+import { MovieList } from './components/MovieList';
+import moviesFromServer from './api/movies.json';
 // import moviesFromServer from './api/movies.json';
```
Comment (Andriy-Shymkiv) [inline src/App.jsx:6]: remove comments

File: src/components/MovieCard/MovieCard.jsx [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={movie.imgUrl}
```
Comment (loralevitska) [inline src/components/MovieCard/MovieCard.jsx:10]: Destructure `movie`

File: src/components/MovieList/MovieList.jsx [hunk 3]
```diff
@@ -1,5 +1,12 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard movie={movie} key={movie.imdbId} />
```
Comment (loralevitska) [inline src/components/MovieList/MovieList.jsx:8]: Destructure `movie`

File: src/components/MovieList/MovieList.jsx [hunk 4]
```diff
@@ -1,5 +1,12 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
```
Comment (Andriy-Shymkiv) [inline src/components/MovieList/MovieList.jsx:5]: redundant fragment

File: src/components/MovieCard/MovieCard.jsx [hunk 5]
```diff
@@ -1,5 +1,43 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
```
Comment (Andriy-Shymkiv) [inline src/components/MovieCard/MovieCard.jsx:4]: redundant fragment

File: src/components/MovieCard/MovieCard.jsx [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+            </p>
+          </div>
+        </div>
+
+        <div className="content">
+          <p data-cy="MovieDescription">
+            {movie.description}
+          </p>
+
+          <a href={movie.imdbUrl} data-cy="MovieLink">
```
Comment (loralevitska) [inline src/components/MovieCard/MovieCard.jsx:36]: Always open links in new tab

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost perfect) Let's improve
- remove the space to refactor link format
image

**Review by TarasHoliuk — APPROVED** [review]

**Pull request closed without merging**